app/controllers/recipe_foods_controller.rb

```diff
@@ -0,0 +1,38 @@
+class RecipeFoodsController < ApplicationController
+  before_action :set_recipe, only: %i[new create update destroy]
+
+  def new
+    @recipe_food = @recipe.recipe_foods.new
+  end
+
+  def create
+    @recipe_food = @recipe.recipe_foods.new(recipe_foods_params)
+
+    respond_to do |format|
+      if @recipe_food.save
+        format.html { redirect_to recipe_url(@recipe), notice: 'Ingredient successfully added.' }
+      else
+        format.html { render :new, status: :unprocessable_entity }
+      end
+    end
```

Comment:
[OPTIONAL] As we are only having one type of response, I would recommend deleting unnecessary code to make it look cleaner.

```suggestion
      if @recipe_food.save
          redirect_to recipe_url(@recipe), notice: 'Ingredient successfully added.'
      else
        render :new, status: :unprocessable_entity
      end
```